core/templates/pages/splash-page/splash-page.component.html

```diff
@@ -182,553 +182,5 @@ <h3 [innerHTML]="'I18N_SPLASH_VOLUNTEERS_TITLE' | translate"></h3>
-    .splash-page .oppia-splash-section-seven-left {
-      margin-left: auto;
-      margin-right: auto;
-      text-align: center;
-    }
-
-    .splash-page .oppia-splash-section-six-right,
-    .splash-page .oppia-splash-section-seven-left {
-      width: 90vw;
-    }
-
-    .splash-page .oppia-splash-section-one-left {
-      margin-left: auto;
-      margin-right: auto;
-      margin-top: 7.5vh;
-      text-align: center;
-      width: 75vw;
-    }
-
-    .splash-page .oppia-splash-section-six-right p,
-    .splash-page .oppia-splash-section-seven-left p {
-      padding-right: 0;
-      text-align: center;
-    }
-
-    .splash-page .oppia-splash-section-five-left {
-      margin-right: 10vw;
-      text-align: center;
-      width: auto;
-    }
-
-    .splash-page .oppia-splash-section-four-left h3,
-    .splash-page .oppia-splash-section-five-left .testimonial-quote-details,
-    .splash-page .oppia-splash-section-six-right h3,
-    .splash-page .oppia-splash-section-seven-left h3 {
-      font-size: 1.5em;
-    }
-
-    .splash-page .splash-student-image img {
-      max-width: 150px;
-    }
-
-    .splash-page .oppia-splash-section-four-left {
-      width: auto;
-    }
-
-    .splash-page .splash-arrows-container {
-      position: relative;
-    }
-
-    .splash-page .oppia-splash-section-four-left .oppia-splash-button,
-    .splash-page .oppia-splash-section-six-right .oppia-splash-button,
-    .splash-page .oppia-splash-section-seven-left .oppia-splash-button {
-      width: 90%;
-    }
-
-    .splash-page .oppia-splash-section-six-left {
-      margin-left: auto;
-      margin-right: auto;
-      width: 70vw;
-    }
-
-    .splash-page .splash-icon-text {
-      display: block;
-      font-size: 0.8em;
-    }
-
-    .splash-page .oppia-splash-icon-1-container {
-      display: block;
-      margin-left: 0;
-    }
-
-    .splash-page .oppia-splash-icon-2-container {
-      display: block;
-    }
-
-    .splash-page .oppia-splash-section-two img {
-      margin-left: 5vw;
-    }
-
-    .splash-page .oppia-splash-icon-3-container {
-      display: block;
-      margin-left: 70vw;
-    }
-
-    .splash-page .oppia-splash-h1,
-    .splash-page .oppia-splash-h2 {
-      display: block;
-      margin-left: 0;
-    }
-
-    .splash-page .oppia-splash-h1 {
-      font-size: 1.7em;
-      line-height: 1.5;
-    }
-
-    .splash-page .splash-benefits-section {
-      margin-left: auto;
-      margin-right: auto;
-      width: 20em;
-    }
-
-    .splash-page .oppia-splash-h2-container {
-      margin-left: 0;
-    }
-
-    .splash-page .oppia-splash-h2-text {
-      font-size: 1.05em;
-      font-weight: 600;
-    }
-  }
-</style>
+<link *ngIf="!isLanguageRTL()" rel="stylesheet" type="text/css" media="screen" href="/templates/pages/splash-page/splash-page.component.css">
+<link *ngIf="isLanguageRTL()" rel="stylesheet" type="text/css" media="screen" href="/templates/pages/splash-page/splash-page.component.rtl.css">
```

Comment:
Hmm, I'm a bit worried about this as we usually don't use direct links to files that are in the templates folder, we usually handle this stuff using webpack. Did you test that this approach works when Oppia is in prod mode and when Oppia is deployed?

Also, would it be possible to handle this using webpack? It doesn't need to be in this PR but it is quite important to handle this in a way that we handle other stuff regarding FE files. TODO would be useful here I think.

Reply:
Ok, I'll add a TODO, and yeah, I tried this locally with `--prod_env` and the server and a page which uses this did load with no issues.

Reply:
Ok, I'll add a TODO, and yeah, I tried this locally with `--prod_env` and the server and a page which uses this did load with no issues.

Reply:
Done

Reply:
We don't use username TODOs anymore, can you please create an issue and add the number here.

Reply:
Looks like this is done.